src/cupbearer/data/tampering.py

```diff
@@ -0,0 +1,62 @@
+from dataclasses import dataclass
+from typing import ClassVar
+
+import torch
+from datasets import load_dataset
+
+from . import DatasetConfig
+
+
+class TamperingDataset(torch.utils.data.Dataset):
```

Comment:
To define a measurement tampering task, I see two approaches (with the task format from #30):
1. Add options to `TamperingDataset` to only return a selection of the data (e.g. only easy training data, or only validation data that has tampering). Then use `Task.from_separate_data()` to recombine them. Advantage is we get some control over mixing ratio and use the standard interface, downside is that it's kind of indirect and complicated.
2. Pass `trusted_data`, `untrusted_train_data`, and `test_data` directly to `Task`, without using `MixedData` at all. This is closer to the format the datasets are already in anyway, so probably simpler. We'll just have to make sure to return the right label format: `trusted_data` and `untrusted_train_data` should just return `(text, measurements)`, whereas `test_data` should return `((text, measurements), is_clean)` I think.

Leaning a bit towards 2. overall but open to either approach or some third one

Reply:
To be clear, no need to do this before merging, just noting down my thoughts